frontend/app/player/MessageDistributor/managers/AssistManager.ts

```diff
@@ -213,6 +213,9 @@ export default class AssistManager {
           }
         }
       })
+      socket.on('videofeed', ({ streamId, enabled }) => {
+        this.videoStreams[streamId].enabled = enabled
```

Comment:
might happen before `call.on('stream'` - then it'll break with an error.

Reply:
all fixed